content/guides/merchant-payment-conditions.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Merchant Payment Conditions
+---
```

Comment:
@tom-blake there is a comment that this guide highlight that it is a payment protocol extension.
Should we add a standard sentence to the top of the description?